### PR TITLE
Django 1.10 compatibility

### PR DESCRIPTION
**Pull request by benred42**
Addresses #102. Tests pass for Django 1.8 with no warnings.

### CODE REVIEW

File: scribbler/templatetags/scribbler_tags.py [hunk 1]
```diff
@@ -173,7 +175,7 @@ def scribble_field(context, model_instance, field_name):
         scribble_template = template.engines['django'].from_string(field_value)
     else:
         scribble_template = template.Template(field_value)
-    scribble_context = build_scribble_context(None, request)
```
Comment (mlavin) [inline scribbler/templatetags/scribbler_tags.py]: This seems problematic since it won't include the variable from the context processors, right?

Comment (benred42) [inline scribbler/templatetags/scribbler_tags.py]: Added the request back in the proper place.  Should fix that problem.

**Comment by mlavin** [comment]
Trying this in the sample project I'm getting CSRF errors trying to the preview. Looks like that is getting lost from the context at some point.

File: scribbler/templatetags/scribbler_tags.py [hunk 2]
```diff
@@ -83,7 +83,9 @@ def render(self, context):
         context['can_edit_scribble'] = can_edit
         context['can_delete_scribble'] = can_delete
         context['raw_content'] = self.raw
-        return wrapper_template.render(context)
+        # render() takes a dict, so we have to extract the context dict from the object
+        context_data = context.dicts[-1]
+        return wrapper_template.render(context_data)
```
Comment (benred42) [inline scribbler/templatetags/scribbler_tags.py]: This is probably what's causing the CSRF error, and again on line 201.  I can extract that information (Along with the request, permissions, etc) from the context object and pass it to render's request parameter and hopefully that'll clear up the CSRF issue.

**Comment by benred42** [comment]
Let me know if you're still getting a CSRF error of if that last commit fixed it.

**Comment by mlavin** [comment]
No more CSRF but preview is still not working
```
Internal Server Error: /scribble/preview/
Traceback (most recent call last):
  File "/home/mlavin/.virtualenvs/django-scribbler/lib/python3.4/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/mlavin/.virtualenvs/django-scribbler/lib/python3.4/site-packages/django/views/decorators/http.py", line 45, in inner
    return func(request, *args, **kwargs)
  File "/home/mlavin/Projects/django-scribbler/scribbler/views.py", line 51, in preview_scribble
    results['variables'] = get_variables(RequestContext(context, request))
  File "/home/mlavin/Projects/django-scribbler/scribbler/utils.py", line 31, in get_variables
    variables = set(context.flatten().keys())
  File "/home/mlavin/.virtualenvs/django-scribbler/lib/python3.4/site-packages/django/template/context.py", line 107, in flatten
    flat.update(d)
ValueError: dictionary update sequence element #0 has length 182; 2 is required
```
This is using Django 1.8.5 and Python 3.4.3 in the example project. Not sure why we aren't catching this in the test suite :\

**Comment by benred42** [comment]
Huh.  I am not getting the same error.  The example project runs normally for me.  Maybe I need to drop the database and re-migrate to get the same error.

**Comment by mlavin** [comment]
:rainbow: :umbrella:

**Pull request merged**